interface/wx/stdpaths.h

```diff
@@ -407,6 +407,17 @@ class wxStandardPaths
     */
     virtual wxString GetUserLocalDataDir() const;
 
+    /**
+        Return OS specific directory where proect shared liraries are
+
+        - Windows: @c "C:\Programs\AppFolder\"
+        - Unix:    @c either /usr/lib/ or /usr/local/lib/
+        - OSX:     @c "/Applications/exename.app/Contents/Frameworks/"
```

Comment:
Maybe clarify that these are examples. I can install somewhere else on Windows. Or run the macOS .app from a different directory.

And maybe use `appinfo` instead of `exename`, like some other documentation does.

Reply:
What version of Windows uses literally `C:\Programs`?

Reply:
@lanurmi ,
I don't know. But that's what the documentation says.

Reply:
> @lanurmi , I don't know. But that's what the documentation says.

I can't find any search results for `C:\Programs` either in 'documentation' or anywhere else on the web.